src/Libraries/Watch3DNodeModelsWpf/HelixWatch3DNodeViewModel.cs

```diff
@@ -74,9 +76,49 @@ protected override void OnModelPropertyChanged(object sender, PropertyChangedEve
             }
         }
 
+        protected override void OnNodePropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            var node = sender as NodeModel;
+            if (node == null)
+            {
+                return;
+            }
+
+            if(e.PropertyName == nameof(node.CachedValue))
+            {
+                var connected = watchModel.InPorts.SelectMany(p => p.Connectors.Select(c => c.Start.Owner));
+                if (!connected.Contains(node))
+                {
+                    return;
```

Comment:
when would this occur?

Reply:
The DefaultWatch is wired up to every node.PropertyChanged event in the graph.  In the case of the background preview you want all of them.  For the Watch3D we want to only react to changes that are for the node that is directly connected